react/package.json

```diff
@@ -69,8 +69,8 @@
         "lodash": "^4.17.21",
         "mathjs": "^9.4.2",
         "nebula.gl": "^0.22.3",
-        "react": "^17.0.2",
-        "react-dom": "^17.0.2",
+        "react": "^16.13.0",
```

Comment:
is it okay to downgrade react? I had to do this install manager-webpack5.
It doesn't get installed with current react version.

Reply:
It's ok, we've updated react with the condition that if anything breaks we'll just downgrade again. Now that something broke, it's alright to get it back to 16.